src/client/session.js

```diff
@@ -0,0 +1,14 @@
+import store from 'store2';
+import { Random } from 'meteor/random';
+
+export function getUserHex() {
+  if (!store.has('user')) {
+    store.set('user', Random.hexString(32));

```

Comment:
noted, and read a bit online subsequently. you probably need a lot of traction to hit a collision with a 32 digit hex. putting up a get request would mean all event submission will need to hold until we get a response back from the server. i would suggest to go with it